app/lib/forms/choose_your_npq.rb

```diff
@@ -2,15 +2,24 @@ module Forms
   class ChooseYourNpq < Base
     include Helpers::Institution
 
-    attr_accessor :course_id
+    QUESTION_NAME = :course_id
```

Comment:
Will this be used for setting up a question and moving to the shared template for the question page? Like in Forms::IttProvider?

Reply:
Yes, it was left over from another branch i have which i removed for this bit of work